controller/embedding/manager.py

```diff
@@ -137,6 +145,63 @@ def get_embedding_name(
     return name
 
 
+EMBEDDING_SCHEMA_WHITELIST = [
+    "id",
+    "name",
+    "custom",
+    "type",
+    "state",
+    "progress",
+    "dimension",
+    "count",
+    "platform",
+    "model",
+    "filter_attributes",
+    "attribute_id",
+    "onQdrant",
```

Comment:
why onQdrant is written with camelcase?

Reply:
falsely added here, not part of the table columns but an added parameter outside the table => removed onQdrant from whitelist